packages/flutter/test/material/icon_button_test.dart

```diff
@@ -3375,6 +3381,34 @@ void main() {
     expect(FocusManager.instance.highlightMode, equals(FocusHighlightMode.traditional));
     expect(inkFeatures, paints..rect(color: focusColor));
   }, skip: !isBrowser); // [intended] tests web-specific behavior.
+
+  testWidgets('IconButton golden test', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('IconButton with a badge', (WidgetTester tester) async {
```

Reply:
@dkwingsmt is it a typo here? `Outlined button's outline` shouldn't this be `IconButton's outline`?

Reply:
@dkwingsmt BTW renamed the test, and added the link to the issue